pkg/services/profiles/update.go

```diff
@@ -101,6 +95,42 @@ func (s *ProfilesSvc) Update(ctx context.Context, gitProvider gitproviders.GitPr
 	return nil
 }
 
+func updatePRInfo(opts UpdateOptions, defaultBranch string, commitFile gitprovider.CommitFile) gitproviders.PullRequestInfo {
```

Comment:
`addPrInfo` is pretty much identical to this one. Can we extract these two into a single function to avoid duplication? 🤔

Reply:
unfortunately they differ enough that its not really practical, different default messages, different struct as an input. I did look into it but didn't seem worth it as it just adds more complexity iumo

Reply:
actually with the `Add/UpdateOptions` struct merging into 1 this is more practical now. I'll refactor it